File: fix_unicode.js
const fs = require('fs');

const file = 'lib/blog/us-manufacturing.ts';
let content = fs.readFileSync(file, 'utf8');

// Replace smart quotes with ASCII
content = content.replace(/\u2019/g, "'")  // '
           .replace(/\u2018/g, "'")  // '
           .replace(/\u2014/g, '--'); // —

fs.writeFileSync(file, content, 'utf8');
console.log('✓ Fixed Unicode characters');

File: fix_unicode.py
import codecs

# Read file in binary mode
with open('lib/blog/us-manufacturing.ts', 'rb') as f:
    content = f.read()

# Replace Unicode smart quotes with ASCII
# \xe2\x80\x99 is UTF-8 for right single quote (')
# \xe2\x80\x98 is UTF-8 for left single quote (')
# \xe2\x80\x94 is UTF-8 for em dash (—)
content = content.replace(b'\xe2\x80\x99', b"'")  # '
content = content.replace(b'\xe2\x80\x98', b"'")  # '
content = content.replace(b'\xe2\x80\x94', b'--')  # —

# Write back
with open('lib/blog/us-manufacturing.ts', 'wb') as f:
    f.write(content)

print('Fixed special characters in us-manufacturing.ts')
